cmd/authgear/migrate.go

```diff
@@ -65,7 +66,16 @@ var cmdMigrateDown = &cobra.Command{
 			log.Fatalf("cannot load secret config: %s", err)
 		}
 
-		migrate.Down(migrate.Options{
+		if len(args) == 0 {
+			log.Fatalf("number of migrations to revert not specified; specify 0 to revert all migrations")
```

Comment:
Maybe support a special value `all` to mean revert all migrations. `authgear migrate down 0` seems a no-op to me, but it is not.